### PR TITLE
Improve portuguese translation

### PR DESCRIPTION
**Pull request by fpaula**

### CODE REVIEW

File: locales/pt.yml [hunk 1]
```diff
@@ -312,8 +312,8 @@ pt:
     page_title: Atualizar Perfil
     subtitle: Crescimento pessoal é uma boa coisa.
     title: O seu Perfil
-    website_url_cant_be_blank: Website URL can't be blank
-    website_url_too_long: Website URL is too long (maximum is %{count} characters)
+    website_url_cant_be_blank: A URL do site não pode ficar em branco
+    website_url_too_long: URL muito longa (maximum is %{count} characters)
```
Comment (jeopard) [inline locales/pt.yml:316]: `maximum is %{count} characters` is missing the translation :)

**Review by jeopard — COMMENTED** [review]
thank you so much for the PR, it looks amazing!! I just had 3 minor comments, I hope they make sense :)

File: locales/pt.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@ pt:
       cta: Enviar e-mail com instruções
       error_forgot_email: Um endereço de e-mail é necessário.
       error_invalid_email: Isso não parece ser um endereço de e-mail válido. Por favor, verifique se está correcto e tente novamente.
-      error_quota: We've recently sent a reset email to the above email address.  It can take up to 5-10 minutes for an email to show up in your inbox.  Please double check that the email did not end up in your spam folder.  If after 10 minutes you have not received the email you may try again to reset your password.
+      error_quota: Nós recentemente enviamos um email de recuperação para o endereço de email abaixo. Pode demorar de 5 a 10 minutos para que o email apareça na sua caixa de entrada. Por favor verifique se seu email não caiu na caixa de spam. Se depois de 10 minutos você não estiver recebido o email você pode tentar redefinir sua senha novamente.
```
[the remark this answers is not in the document]
Comment (fpaula) [reply locales/pt.yml:63]: My mistake @jeopard. Above means the opposite of that lol.

[the remark this answers is not in the document]
Comment (fpaula) [reply locales/pt.yml:63]: @jeopard about "you have not received", it is correct in my opinion.

[the remark this answers is not in the document]
Comment (jeopard) [reply locales/pt.yml]: thank you for the PR! :)

**Pull request merged**